server/rtcd.go

```diff
@@ -706,3 +708,51 @@ func (h *rtcdHost) isFlagged() bool {
 	defer h.mut.RUnlock()
 	return h.flagged
 }
+
+func (m *rtcdClientManager) hasCallEnded(call *public.Call) bool {
+	m.ctx.LogDebug("RTCD host is set in call, checking...", "callID", call.ID, "rtcdHost", call.Props.RTCDHost)
+	host := m.getHost(call.Props.RTCDHost)
+	if host != nil {
+		m.ctx.LogDebug("RTCD host found", "callID", call.ID, "rtcdHost", call.Props.RTCDHost)
+
+		// Version compatibility check. We need to be talking to RTCD v1.0.0 or higher to be able to call GetSessions.
+		info, err := host.client.GetVersionInfo()
+		if err != nil {
+			m.ctx.LogDebug("failed to get version info", "err", err.Error(), "callID", call.ID, "rtcdHost", call.Props.RTCDHost)
+			return false
```

Comment:
This defaults to call is still running if there's a problem getting the version info -- will that cause trouble? Should we return the err and have logic for it?